package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "dan": "tsx ./scripts/release-unpublish.ts",
     "dev": "nodemon --config nodemon.config.json -x 'pnpm build:packages'",
-    "build": "turbo run build",
+    "build": "turbo run build --cache-dir=.turbo",
```

Comment:
I experienced several cache invalidation issues with Turbo Repo while working on Ubuntu. 

Do you think these issues might be solved with the changes introduced in this pull request?

Reply:
@danielbate Not sure if it is related, but something [broken](https://github.com/FuelLabs/fuels-ts/actions/runs/7556364202/job/20573212449?pr=1649).

Reply:
Hmm it should still be caching from `node_modules` by default, this just makes the path more easily accessible from CI. Let me know if this does solve it for you.

Reply:
On second thoughts, we should not be using partial restores here. Unless anyone can think of a reason to? I believe that is why the build was failing. Have resolved it by removing `restore-keys`. Worked the first time round as it was the first cache run. We can see it working in a dependent stage [here](https://github.com/FuelLabs/fuels-ts/actions/runs/7560466639/job/20586820316#step:3:153).